Library/Formula/gobject-introspection.rb

```diff
@@ -16,7 +16,9 @@ class GobjectIntrospection < Formula
 
   depends_on "pkg-config" => :run
   depends_on "glib"
+  depends_on "cairo"
   depends_on "libffi"
+  depends_on :python
```

Comment:
`depends_on "python" => :build if MacOS.version == :mavericks` is probably fine for now.